_data/meltano/utilities/evidence/meltanolabs.yml

```diff
@@ -0,0 +1,256 @@
+commands:
+  describe:
+    args: describe
+    description: Describe the Evidence extension.
+    executable: evidence_extension
+  initialize:
+    args: initialize
+    description: |
+      Initialize a new Evidence project. By default this creates a sample project in `$MELTANO_PROJECT_ROOT/analyze/evidence`.
+    executable: evidence_extension
+definition: |
+  is a Meltano utility extension that enables you to deliver a polished business intelligence system using SQL and markdown.
+description: |
+  Business Intelligence for Modern Data Teams.
+  Beautiful reports synced to your data with just SQL and markdown.
+docs: https://docs.evidence.dev
+domain_url: https://evidence.dev
+executable: evidence_extension
+ext_repo: https://github.com/MeltanoLabs/evidence-ext
+keywords:
+- meltano_edk
+- evidence.dev
+- evidence
+- bi
+- analyze
+label: Evidence (EDK)
+logo_url: /assets/logos/utilities/evidence.png
+maintenance_status: active
+name: evidence
+namespace: evidence
+next_steps: |
+  *Create a new Evidence project*
+
+    1. Run `meltano invoke evidence initialize` to initialize Evidence in the default location (`analyze/evidence`).
+    1. Run `meltano invoke evidence dev` to see the sample project site (includes a DuckDB database and config.)
+    1. Configure your own database by:
+      - Setting the database connection type, e.g. `meltano config evidence set database snowflake` for Snowflake.
```

Comment:
```suggestion
      - Setting the database connection type, e.g. `meltano config evidence set settings.database snowflake` for Snowflake.
```

Reply:
@pnadolny13 these are two different settings 😅 Evidence itself requires a `database` to be selected (one of `duckdb`, `sqlite`, `snowflake` etc.) _and_ Snowflake itself requires a database (`setting.snowflake.database`). This is because Evidence supports multiple databases, so needs to know which one is 'enabled' and therefore which config env vars to read 👍